views/artist-search-result.hbs

```diff
@@ -0,0 +1,9 @@
+{{#if artists}}
+    {{#each artists}}
```

Comment:
I am not 100% sure, but I think #each doesn't do anything if there is nothing to #each over... so it should be fine to drop that 1 level of nesting I think.